.github/workflows/post-apiview.yml

```
@@ -0,0 +1,32 @@
+name: After APIView
+
+on:
+  check_run:
+    types: [completed]
+    
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  post-apiview:
+    name: After APIView
+    runs-on: ubuntu-24.04
+    if: |
+      ${{ github.event.check_run.check_suite.app.name == 'Azure Pipelines' && 
+      ( contains(github.event.check_run.name, 'APIView') ||
+      contains(github.event.check_run.name, '(SDK Generation') ||
+      contains(github.event.check_run.name, 'Azure.azure-rest-api-specs-pipeline') ) }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: 'eng/common'
+    
+      - name: Create APIView Comment on PR
+        run: |
+          . "${{ github.workspace }}/eng/common/scripts/Helpers/ApiView-Helpers.ps1"
```

```suggestion
          . eng/common/scripts/Helpers/ApiView-Helpers.ps1
```

Are you sure you need to use an absolute path?  If relative path works, I think it's shorter and simpler.

---

This work in my testing.

---

Yes, both paths will work.  Why do you want to use the longer path with more code?